LayoutTests/fast/css/background-position-serialize.html

```diff
@@ -99,6 +99,41 @@
 shouldBe("t.style.cssText", "background-position: 20% 20%;");
 shouldBe("t.getAttribute('style')", "background-position: 20% 20%;");
 
+run("t.style.backgroundPositionY = '0%';");
```

Comment:
Testcases should be added in this WPT as well: css/css-backgrounds/parsing/background-position-valid.html